apps/nuxt3-ssr/components/content/ContentBlockAttachedFiles.vue

```diff
@@ -16,6 +16,10 @@ function looksLikeImage(document: IDocumentation) {
   );
 }
 
+function isExternalDocument(document: IDocumentation) {
+  return document?.file.url ? false : true;
+}
+
```

Comment:
minor: wouldn't a computed be better? Since this way it's executed every re-render.